tests/run-browser.mjs

```diff
@@ -163,4 +174,30 @@ function logIncrementalResult(previousResults, benchmarkResults) {
     }
 }
 
+
+function printLogs(sessionId) {
+    if (BROWSER === "safari" && sessionId)
+        return printSafariLogs(sessionId);
+
+}
+
+async function printSafariLogs(sessionId) {
```

Comment:
Is this logging required for making it work, or is this just for debugging failures in the future?